src/apis/oracle.ts

```diff
@@ -70,7 +72,42 @@ export class DefaultOracleAPI implements OracleAPI {
         }
         const encodedExchangeRate = encodeUnsignedFixedPoint(this.api, exchangeRate);
         const tx = this.api.tx.exchangeRateOracle.setExchangeRate(encodedExchangeRate);
-        const result = await sendLoggedTx(tx, this.account, this.api);
```

Comment:
This should not be removed. You can remove the left hand side of the assignment since it's not used, but we need the `await sendLoggedTx(tx, this.account, this.api);`